src/mots/utils.py

```diff
@@ -34,13 +34,12 @@ def get_list_input(text: str):
 
 def parse_real_name(real_name):
     """Parse real_name into name and info."""
-    pattern = re.compile(r"^(?P<name>[\w\ ]+?)?\ ?(?P<info>[\(\[\|\:].*)?$")
+    pattern = re.compile(r"^(?P<name>[\w\ \-]*)?\ ?(?P<info>\W.*)?$")
```

Comment:
Nit: this should be compiled this outside the function since it doesn't rely on any of the arguments.